model.md

```diff
@@ -0,0 +1,66 @@
+# Pedigree conceptual model
+
+## Motivation
+
+Lack of interoperability of pedigree data between systems results in a substantial double-entry burden for already-overloaded nurses, genetic counselors, and support staff, and introduces the potential for serious errors.
+
+# Classes
+
+![image](https://user-images.githubusercontent.com/4251264/105409205-1c53b200-5bfe-11eb-9d77-ed5630138eaf.png)
+
+## Pedigree
+
+A `Pedigree` is collection of selected information about a family, including the individuals, relationships between them, and relevant health conditions.
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| proband | ID | optional | id of Individual that is the index case for the family |
+| consultand | ID | optional | id of Individual that is the focus of the current analysis |
+| collectedAt | Date | optional | the date the pedigree was collected, as ISO full or partial date, i.e. YYYY, YYYY-MM, or YYYY-MM-DD |
+| reasonCollected | Concept | optional | the health condition being investigated in the family; if any Individual has the `affected` property defined, it refers to this condition |
+
+## Individual
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| id | ID | required | logical id |
+| name | string | optional | name of the individual |
+| identifiers | list of string | optional | external identifiers for the individual |
+| sex | Concept | optional | Sex for Clinical Use, values: `male`, `female`, `other`, `unknown`; See: https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions and https://www.hl7.org/fhir/valueset-administrative-gender.html |
+| gender | Concept | optional | Presumed or reported gender identity, values: `male`, `female`, `non-binary`, `non-disclosed`, `trans`, ...; See: https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions |
+| lifeStatus | Concept | recommended | The presumed/accepted life status of the individual as of the pedigree collection date; one of: `alive`, `deceased`, `unborn` |
+| affected | boolean | not recommended | whether or not the individual is affected by the condition being investigated in this pedigree; included for PED backwards compatibility |
```

Comment:
I agree that the affection status had better be kept at `Individual` since it's not an at-a-point-in-time characteristic. Re the life status, as commented here (https://github.com/ga4gh-cp/pedigree/issues/7), I am not sure if it should be kept at Individual for pedigree information.

Reply:
I agree in principle, but I think the complexity introduced by having point-in-time observations is one that we should wait to introduce until the next version. Based on our discussion, I thought having a static representation of life/vital status for v1 would be better than leaving it out entirely until v2. If you'd prefer to remove it from this version, happy to discuss and decide on the call today!